README.md

```diff
@@ -1,97 +1,115 @@
-[KIND](https://sigs.k8s.io/kind) to run a local cluster for testing, or you 
-can run against a remote cluster.  
-
-**Note:** Your controller will automatically use the current context in your 
-kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
-
-### Start a Development Cluster
+Armada Operator is a Kubernetes-native Operator for simpler installation of [Armada](https://armadaproject.io).
+This project introduces CRDs for Armada services and provides a controller to manage the lifecycle of these services.
 
-This section assumes you have [KIND](https://sigs.k8s.io/kind) installed.
+## How it works
+This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
 
-If you do not have a Kubernetes cluster to test against, you can start one using the following command:
-```bash
-make create-dev-cluster
-```
+It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
+which provides a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster
 
-### Install using Helm
+## Installation
 
-First, add the `gresearch` public Helm registry:
-```bash
-helm repo add gresearch https://g-research.github.io/charts
-```
+You’ll need a Kubernetes cluster to run the Armada Operator. You can use
+[KIND](https://sigs.k8s.io/kind) to run a local cluster for testing, or you
+can run against a remote cluster.
 
-After that, you can install the Armada Operator using Helm:
+To install the latest release of Armada Operator in your cluster, run the following commands:
 ```bash
+# Add the G-Research Helm repository
+helm repo add gresearch https://gresearch.github.io/armada-operator
+# Install the Armada Operator
 helm install armada-operator gresearch/armada-operator --namespace armada-system --create-namespace
 ```
 
-### Build & install from scratch
+The Operator will run in the `armada-system` namespace.
 
-Run the following command to install all Armada external dependencies (Apache Pulsar, Redis, PostgreSQL, NGINX, Prometheus)
-```bash
-make dev-setup
-```
+### Installing Armada
 
-Then:
+In order to install Armada, first you will need to install the Armada external dependencies:
+* [Pulsar](https://pulsar.apache.org/)
+* [Redis](https://redis.io/)
+* [Postgres](https://www.postgresql.org/)
+
+You can install the required dependencies either manually or by running the following command:
 ```bash
-make dev-install-controller
+make install-armada-deps
 ```
-Which will:
-- install each CRD supported by the armada-operator on the cluster
-- create a pod inside the kind cluster running the armada-operator controllers
 
-**Note:** You may need to wait for some services (like Pulsar) to finish 
-coming up to proceed to the next step. Check the status of 
-the cluster with `$ kubectl get -n armada pods`.
+**Note:** You may need to wait for some services (like Pulsar) to finish
```

Comment:
You will need to wait for all the services (Pulsar, Postgres, etc)  to start before proceeding to the next step.

Reply:
resolved